aten/src/ATen/cpu/vec/sve/vec_int.h

```diff
@@ -4,8 +4,8 @@
 #include <ATen/cpu/vec/vec_base.h>
 #include <ATen/cpu/vec/sve/sve_helper.h>
 
-namespace at {
-namespace vec {
+
+namespace at::vec {
```

Comment:
This is modernize namespace, not performance. Are you sure you selected the rules correctly @cyyever ?

Reply:
modernize-* also enabled, let me exclude them.